test/python/test_kinetics.py

```diff
@@ -1924,3 +1932,16 @@ def test_electron_collision_plasma(self):
                                                           electron_energy_dist)
         k2 = gas2.forward_rate_constants[1]
         assert k1 == approx(k2)
+
+    def test_ECPR_rate_of_progress(self):
+        # Test of forward and reverse rate of progress
+        # The reverse rate of progress calculates the super-elastic rate coefficient
+        # which equals to the forward rate coefficient when the threshold energy is
+        # zero. Therefore, the reverse rate of progress equals the forward rate of
+        # progress.
+        gas = self.setup_plasma()
+
+        frop = gas.forward_rates_of_progress[1]
+        rrop = gas.reverse_rates_of_progress[1]
+
+        assert frop == rrop
```

Comment:
Even if this should be exactly equal, you probably still want to use `approx` for the floating point comparison.